main.go

```diff
@@ -98,6 +98,7 @@ func setupFlags(cmd *cobra.Command) {
 	flagSet.StringVar(&cfg.ImageProxy, "image-proxy", "", "Http proxy to pull image")
 	flagSet.StringVar(&cfg.QuotaDriver, "quota-driver", "", "Set quota driver(grpquota/prjquota), if not set, it will set by kernel version")
 	flagSet.StringVar(&cfg.ConfigFile, "config-file", "/etc/pouch/config.json", "Configuration file of pouchd")
+	flagSet.StringVar(&cfg.Snapshotter, "snapshotter", "overlayfs", "Snapshotter of pouchd")
```

Comment:
Could we make the description more detailed? @wangforthinker 

Set the snapshotter driver name of containerd?

Reply:
Have Done it!